tests/test_rates_at_checkout.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+import shippo
+from shippo.models.components import ServiceGroupCreateRequest, Carriers, \
+    ServiceGroupAccountAndServiceLevel, ServiceGroupType, LiveRateCreateRequest, AddressCompleteCreateRequest, \
+    LineItem, WeightUnit, Parcel, DistanceUnitTemplate
+from tests.helpers_custom import get_carrier_account
+
+
+class TestRatesAtCheckout:
+
+    @pytest.fixture(scope='class', autouse=True)
+    def setup(self, api: shippo.Shippo):
+        self._delete_all_service_groups(api)
```

Comment:
why is this required?

Reply:
because service group creation fails if you try to create two service groups with the same name and I'd rather no have hundreds or thousands of service groups with the same definition and different names since the service groups are used internally when generating rates